.expeditor/scripts/chef-infra-client.spec

```diff
@@ -0,0 +1,89 @@
+# Disable any shell actions, replace them with simply 'true'
+%define __spec_prep_post true
+%define __spec_prep_pre true
+%define __spec_build_post true
+%define __spec_build_pre true
+%define __spec_install_post true
+%define __spec_install_pre true
+%define __spec_clean_post true
+%define __spec_clean_pre true
+
+# Use SHA256 checksums for all files
+%define _binary_filedigest_algorithm 8
+
+%define _binary_payload w1.xzdio
```

Comment:
We might want to disable compresssion - it will slow down the build without much advantage, because we're primarily shipping two already-compressed tarballs.